model/src/pyrenew/model/rtinfectionsrenewalmodel.py

```diff
@@ -284,6 +285,14 @@ def sample(
         # Sampling initial infections
         i0, *_ = self.sample_i0(**kwargs)
 
+        # Padding i0 to match gen_int
+        # PADDING SHOULD BE REMOVED ONCE
+        # https://github.com/CDCgov/multisignal-epi-inference/pull/124
+        # is merged.
+        # SEE ALSO:
+        # https://github.com/CDCgov/multisignal-epi-inference/pull/123#discussion_r1612337288
+        i0 = du.pad_x_to_match_y(x=i0, y=gen_int, fill_value=0.0)
```

Comment:
Not doing so breaks the tests as we pass a scalar in some places (tests and examples). I could edit all places where that happens, but it would be inefficient as #124 solves this internally.